lib/packagekit-glib2/pk-progress-bar.c

```diff
@@ -131,10 +133,13 @@ pk_progress_bar_draw (PkProgressBar *self, gint percentage)
 	for (i = 0; i < self->priv->size - section; i++)
 		g_string_append (str, " ");
 	g_string_append (str, "] ");
-	if (percentage >= 0 && percentage < 100)
-		g_string_append_printf (str, "(%i%%)  ", percentage);
-	else
+	if (percentage >= 0 && percentage < 100) {
+		/* TRANSLATORS: this is a percentage value we use in messages, e.g. "90%. */
+		g_string_append_printf (str, _("(%i%%)"), percentage);
```

Comment:
do we need `%i` (signed) format specifier ?

Reply:
Replaced with `%d`.

Reply:
Either would have been fine here, when used in printf() there's no real difference between them anymore.